streams/src/main/java/org/apache/kafka/streams/state/Stores.java

```diff
@@ -62,9 +60,9 @@ public static StoreFactory create(final String name) {
                                     @Override
                                     public StateStoreSupplier build() {
                                         if (capacity < Integer.MAX_VALUE) {
-                                            return new InMemoryLRUCacheStoreSupplier<>(name, capacity, serdes, null);
+                                            return new InMemoryLRUCacheStoreSupplier<>(name, capacity, keySerde, valueSerde, null);
                                         }
-                                        return new InMemoryKeyValueStoreSupplier<>(name, serdes, null);
+                                        return new InMemoryKeyValueStoreSupplier<>(name, keySerde, valueSerde, null);
```

Comment:
`null` seems like a magic value to me here.  Should we define sth like a constant `IGNORE_TIME = null` and pass it in here to clarify the meaning?